views/blacklist.ejs

```diff
@@ -1,278 +1,64 @@
-                    <div class="table-quote w-50 me-1 p-2">It's the tree. Gandalf! Gandalf! </div>
-                    <div class="table-comment w-50 p-2 ms-1"><span class="text-decoration-underline">Jouw
-                            opmerking:</span>&nbsp; Pippin is stating the obvious here, and we all know that that is
-                        Legolas' role in the movies. They're taking the Hobbits to Isengard, to Isengard, to
-                        Isengard...</div>
-                    <div class="table-icons me-auto">
-                        <div class="table-edit-button ms-end p-2"><button type="button" data-bs-toggle="collapse"
-                                data-bs-target="#comment3" aria-expanded="false" aria-controls="comment3"
-                                class="btn btn-outline-primary" title="Wijzig deze opmerking"><i
-                                    class="bi bi-pencil-square"></i></button></div>
-                        <div class="table-delete-button ms-end p-2">
-                            <button type="button" class="btn btn-outline-primary" title="Verwijder uit je blacklist"
-                                data-bs-toggle="modal" data-bs-target="#deleteBlackList3">
-                                <i class="bi bi-trash"></i>
-                            </button>
-                        </div>
-                    </div>
-                    <div class="modal fade" id="deleteBlackList3" tabindex="-1" aria-labelledby="deleteBlackList3"
-                        aria-hidden="true">
-                        <div class="modal-dialog modal-dialog-centered">
-                            <div class="modal-content">
-                                <div class="modal-header">
-                                    <h2 class="modal-title fs-5" id="deleteBlackList3">Weet je zeker dat je deze
-                                        quote uit je blacklist wil verwijderen?</h2>
-                                </div>
-                                <div class="modal-body">
-                                    <p>It's the tree. Gandalf! Gandalf!</p>
-                                </div>
-                                <div class="modal-footer">
-                                    <button type="button" class="btn btn-secondary" data-bs-dismiss="modal"
-                                        aria-label="Close">Behouden</button>
-                                    <button type="button" class="btn btn-primary" formmethod="POST"
-                                        formaction="/">Verwijderen</button>
-                                </div>
+                        <!-- Confirm delete window -->
+                        <form action="/blacklist/<%= item.quote_id %>/delete" method="post">
+                            <div class="modal fade" id="deleteBlackList<%= index + 1 %>" tabindex="-1"
+                                aria-labelledby="deleteBlackList<%= index + 1 %>" aria-hidden="true">
```

Comment:
You accidentally deleted part of the "confirm delete window", here is the complete div again, or refer back to the original code from before

```suggestion
                    <!-- Confirm delete window -->
                    <div class="modal fade" id="deleteBlackList1" tabindex="-1" aria-labelledby="deleteBlackList1"
                        aria-hidden="true">
                        <div class="modal-dialog modal-dialog-centered">
                            <div class="modal-content">
                                <div class="modal-header">
                                    <h2 class="modal-title fs-5" id="deleteBlackList1">Weet je zeker dat je deze
                                        quote uit je blacklist wil verwijderen?</h2>
                                </div>
                                <div class="modal-body">
                                    <p>Yes the white tree of Gondor. The tree of the King. Lord Denethor
                                        however, is not the King. He is a steward only, a caretaker of the
                                        throne.</p>
                                </div>
                                <div class="modal-footer">
                                    <button type="button" class="btn btn-secondary" data-bs-dismiss="modal"
                                        aria-label="Close">Behouden</button>
                                    <button type="button" class="btn btn-primary" formmethod="POST"
                                        formaction="/">Verwijderen</button>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>

```